draft-normington-jsonpath.md

```diff
@@ -188,7 +196,11 @@ Output Path:
 Position:
 : A JSON data item identical to or nested within the JSON data item to
   which the query is applied to, expressed either by the value of that
-  data item or by providing a JSONPath Output Path.
+  data item or by providing a Normalized Path Expression as a JSONPath Output Path.
+
+Normalized Path Expression:
+: A query in a normalized form that identifies exactly one Position in
```

Comment:
Normalized form is not yet well defined. Is it intended to be [canonical](https://en.wikipedia.org/wiki/Canonical_form) too or simply normalized (but not necessarily unique among normalized alternatives)?

Reply:
No, that's why it points to a section that explains in which way this is normalized.